chia/seeder/dns_server.py

```diff
@@ -69,18 +71,25 @@ class DNSServer:
     lock: asyncio.Lock
     pointer: int
     crawl_db: aiosqlite.Connection
+    prometheus: PrometheusSeeder
 
-    def __init__(self):
+    def __init__(self, config: Dict, root_path: Path):
         self.reliable_peers_v4 = []
         self.reliable_peers_v6 = []
         self.lock = asyncio.Lock()
         self.pointer_v4 = 0
         self.pointer_v6 = 0
-        db_path_replaced: str = "crawler.db"
-        root_path = DEFAULT_ROOT_PATH
-        self.db_path = path_from_root(root_path, db_path_replaced)
+
+        if "crawler_db_path" in config and config["crawler_db_path"] != "":
+            path = Path(config["crawler_db_path"])
+            self.db_path = path.expanduser()
```

Comment:
`path_from_root` requires this to be somewhere within the CHIA_ROOT though right? That doesn't work specifically for the case this was added to solve, when we need to mount a k8s volume into the container, which happens prior to the chia root existing / chia init being run - for example, we usually just make this path `/crawler-db` inside the container, and that is backed by a persistent volume,  but doesn't need to worry about all the other chia config that change depending on the other flags to the container. 

TBH, I'm not sure if there is really much of a use case where we need to change the path to the crawler DB if its going to be inside of the chia root. I just need a way I can be like "hey crawler, your DB lives over here, instead of the chia root, because of this weird k8s volume thing we're dealing with"

Reply:
Note lines 13-14 where it only resolves relative to `root` if the path is not already absolute.  It also expands `~` on line 11.  So you can specify an absolute path, a user-dir absolute/relative/whatever-you-call-it path, or chia root relative path.

Reply:
I see - I missed that reference snippet you had here last time - I get what you're saying now

Reply:
Simplified this to default crawler_db_path to `crawler.db` and tested that absolute paths work in there as expected